sp/src/game/server/hl2/npc_citizen17.cpp

```diff
@@ -3246,7 +3263,9 @@ Activity CNPC_Citizen::NPC_TranslateActivity( Activity activity )
 
 #ifdef EZ2
 	// Unarmed citizens use 'panic readiness' activities!
-	if ( (!m_bWillpowerDisabled && GetActiveWeapon() == NULL && m_NPCState == NPC_STATE_COMBAT) /*|| IsSurrendered()*/ )
+	// Only if they have no weapon and either cannot throw grenades or are panicking, and are not jump rebels 
+	bool disarmed = GetActiveWeapon() == NULL && m_Type != CT_LONGFALL && ( HasCondition( COND_CIT_WILLPOWER_VERY_LOW ) || !IsGrenadeCapable() || m_iNumGrenades == 0 );
+	if ( (!m_bWillpowerDisabled && disarmed && m_NPCState == NPC_STATE_COMBAT) /*|| IsSurrendered()*/ )
```

Comment:
I don't know if this is worth mentioning, but might not directly cooperate with `ThrowGrenadeAtTarget` in maps because that input ignores the number of grenades a NPC has. I'm assuming this isn't setting out to do anything with that anyway though.

Reply:
Hmm... I think that input should always force a grenade, right? It's a valid point but I'm not sure how to work around it. The intended behavior of that input is to force a grenade throw even if the NPC has no grenades